.dotnet/src/Custom/Chat/ChatCompletionOptions.Serialization.cs

```diff
@@ -13,7 +13,7 @@ public partial class ChatCompletionOptions
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     private void SerializeMessagesValue(Utf8JsonWriter writer, ModelReaderWriterOptions options)
     {
-        if (Messages is not null)
```

Comment:
Should this be reverted now after the other change?

Reply:
nitpicking: CodeGen uses "!=" instead of "is not" 😋

Reply:
good call -- this is now spurious both in its current and previous form. We're guaranteed it's always non-null, so we can get rid of the check and "else" entirely (the else block wouldn't even produce valid-to-schema output anyhow)